apps/cookbook/src/app/showcase/grid-showcase/grid-showcase.component.ts

```diff
@@ -7,6 +8,21 @@ import { ApiDescriptionProperty } from '~/app/shared/api-description/api-descrip
   templateUrl: './grid-showcase.component.html',
 })
 export class GridShowcaseComponent {
+  breakpoints = DesignTokenHelper.breakpoints;
+  breakpointKeyToShorthand = new Map<string, string>([
```

Comment:
Do we need a map or could we simply use a typed object literal? 🤔 
```suggestion
  breakpointKeyToShorthand: { [Property in (keyof typeof styles.breakpoints)]: string }
```